src/test/java/com/google/step/youtube/YouTubeInfoScraperTest.java

```diff
@@ -257,4 +271,27 @@ public void scrapePromoCodesFromPlaylist_IOException() throws IOException {
         when(mockListPlaylistItems.execute()).thenThrow(IOException.class);
         assertThrows(IOException.class, () -> scraper.scrapePromoCodesFromPlaylist(UPLOAD_ID));
     }
+
+    @Test
+    public void scrapeVideoInformation_returnNull() throws IOException {
+        VideoListResponse testVideoResponse = new VideoListResponse();
+        when(mockListVideos.execute()).thenReturn(testVideoResponse);
+        Optional<List<Video>> actual = scraper.scrapeVideoInformation(Arrays.asList(NONEXISTENT_VIDEO_ID));
+        assertThat(actual.isPresent(), equalTo(false));
+    }
+
+    @Test
+    public void scrapeVideoInformation_standardRequest() throws IOException {
```

Comment:
nit: More informative to put "_succeeds" or "_validRequest" rather than "_standardRequest"